src/util/ajax.js

```diff
@@ -354,7 +354,9 @@ export const getImage = function(requestParameters: RequestParameters, callback:
         if (err) {
             callback(err);
         } else if (data) {
-            if (window.createImageBitmap) {
+            // force Safari to use the Image API to avoid a potential crash
+            // See https://github.com/mapbox/mapbox-gl-js/issues/11494
+            if (!isSafari(window) && window.createImageBitmap) {
```

Comment:
Is it possible for this type of check to be more granular and check for version number in the user agent?
e.g. `isSafari(window, '15.3'))`. If this is fixed in 15.4, maybe it wouldn't make sense to prevent this code path for all future Safari versions, so users can still benefit from https://github.com/mapbox/mapbox-gl-js/issues/10491.